apps/src/templates/VersionHistory.jsx

```diff
@@ -183,6 +185,12 @@ export default class VersionHistory extends React.Component {
               versionId={version.versionId}
               lastModified={new Date(version.lastModified)}
               isLatest={version.isLatest}
+              isViewingVersion={
+                this.props.viewingVersion
+                  ? version.versionId === this.props.viewingVersion
+                  : version.isLatest
+              }
+              isProjectOwned={this.props.isProjectOwned}
```

Comment:
(minor - naming) something like `isEditable` or `canChangeVersion` may more clearly represent what the effect of the prop is in the component.

Reply:
I agree that there might be a better name for this... I was thinking about canEditProject, but now I can think of other (potentially relevant in the future) reasons why a person might be able to/not be able to edit a project. Is it already turned in, is it part of a portfolio, is it locked, etc. What do you think about something like "isProjectEditor"? So the emphasis is still on the role of the viewer and couldn't be confused with some of these other use cases?

Reply:
Okay cool- done